packages/editor/src/components/preview-dropdown/index.js

```diff
@@ -62,6 +64,61 @@ export default function PreviewDropdown( { forceIsAutosaveable, disabled } ) {
 		desktop,
 	};
 
+	/**
+	 * The choices for the device type.
+	 *
+	 * @type {Array}
+	 */
+	const choices = [
+		{
+			value: 'Desktop',
+			label: __( 'Desktop' ),
+			icon: desktop,
+		},
+		{
+			value: 'Tablet',
+			label: __( 'Tablet' ),
+			icon: tablet,
+		},
+		{
+			value: 'Mobile',
+			label: __( 'Mobile' ),
+			icon: mobile,
+		},
+	];
+
+	/**
+	 * The selected choice.
+	 *
+	 * @type {Object}
+	 */
+	let selectedChoice = choices.find(
+		( choice ) => choice.value === deviceType
+	);
+
+	/**
+	 * If no selected choice is found, default to the first
+	 */
+	if ( ! selectedChoice ) {
+		selectedChoice = choices[ 0 ];
+	}
+
+	/**
+	 * Handles the selection of a device type.
+	 *
+	 * @param {string} value The device type.
+	 */
+	const onSelect = ( value ) => {
+		setDeviceType( value );
+		if ( value === 'Desktop' ) {
+			speak( __( 'Desktop selected' ), 'assertive' );
+		} else if ( value === 'Tablet' ) {
+			speak( __( 'Tablet selected' ), 'assertive' );
+		} else {
+			speak( __( 'Mobile selected' ), 'assertive' );
+		}
```

Comment:
Thanks for this PR! These announcements shouldn't be necessary. Could you remove these in a follow-up?

Reply:
@jeryj Sure

Reply:
@jeryj created PR to remove `speak` from menu selection.
https://github.com/WordPress/gutenberg/pull/64115